presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -389,10 +388,7 @@ public static void ensurePathExists(Path path)
 
     public void tryStop()
     {
-        if (!isRunning()) {
-            log.warn("Could not stop already stopped container: %s", logicalName);
```

Comment:
Have you seen this message logged?

Reply:
Nope. And it's already guarded inside the container which checks whether the container was already stopped.